### DISCUSSION TITLE
Can I talk to you about Rumbas?

### DISCUSSION THREAD

**Discussion by christianp** [m8rex/rumbas#56] [General]
@jhoobergs , you've done an amazing amount of work making rumbas. 

I'm starting to write some essays about the design of Numbas, and while sketching out a piece on the editing interface, I remembered about this project. I'd love to talk to you about writing questions as plain text files, and what you think the benefits and drawbacks are compared to the web editor.

Are you available for a video call any time?

**Comment by jhoobergs** [comment]
@christianp , thanks for the complement. Rumbas wouldn't exist without Numbas ofcourse :)

I would love to talk to you about why I created rumbas instead of just using Numbas. 

The next two weeks I'm free most of the time, so you can suggest some moments that would work for you. My timezone is UTC+2 so we will need to take that into account :)
